MetaMask/metamask-mobile
# fix: push notifications

## **Description**

This PR re-enable push notifications (background/foreground) based on notifee approach.

## **Related issues**

Fixes:

[NOTIFY-1103](https://consensyssoftware.atlassian.net/issues/?jql=project+%3D+%22NOTIFY%22+AND+assignee+%3D+712020%3A5eed27f4-5f0b-4c08-92a0-9879b5f69a8d+ORDER+BY+created+DESC&atlOrigin=eyJpIjoiYmFjNDg1NDY1Y2ZlNDRiMGEwNTI0N2MzNzcxZjlmZjkiLCJwIjoiaiJ9)
## **Manual testing steps**

1. Install the MetaMask wallet
2. Click on the Bell Icon on the top-right header
3.Turn On Notifications
4. Trigger any transaction and leave the app.

Expected behaviour: A device push notification should appear on your device.

## **Screenshots/Recordings**



### **Before**



### **After**

https://github.com/user-attachments/assets/59938cbf-4a2c-4409-9b68-085ee67dcc9d




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**



[NOTIFY-1103]: https://consensyssoftware.atlassian.net/browse/NOTIFY-1103?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
